.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+*Brief description of the business purpose and effect of the pull request.*
+
+TODO.
+
+# Reference
+
+*List of relevant links to GitHub issues, PRs, and other documentation.*
+
+1. TODO.
+
+# Testing
+
+*Brief description of the testing done for this change showing why you are confident it works as expected and does not introduce regressions. Provide sample output data where appropriate.* 
+
+TODO.
+
+# Checklist
+
+*Checklist of tasks commonly-associated with schema pull requests. Please review the relevant checklists and ensure you do all the tasks that are required for the change you made.*
+
+1. [ ] Add relevant examples.
+2. [ ] Add relevant counterexamples.
+3. [ ] Update in-schema documentation using plain English written in complete sentences, if an update is required.
+4. [ ] Update Docusaurus documentation, if an update is required.
+5. [ ] Review change with Overture technical writer to ensure any advanced documentation needs will be taken care of, unless the change is trivial and would not affect the documentation.
+
+# Documentation Website
+
+*Update the hyperlink below to put the pull request number in.*
+
```

Comment:
Should we explain this can only be edited after PR is created?

Reply:
See how it plans out, and might be worth an effort in an update?